app/js/controllers/wallDisplay.js

```diff
@@ -43,8 +43,8 @@ function WallDisplay($scope, $stateParams, $interval, $timeout, $location, $http
         	}
         }
 
-        if (vm.wallOptions.layoutStyle === '4') {
-            if (term === "") {
+        if (vm.wallOptions.layoutStyle == '4') {
+            if (term == "") {
```

Comment:
@sudheesh001 agreed. However, using `===` seems to be breaking things. Till we figure exactly what is breaking, we can proceed with this.